### ISSUE TITLE
Data Augmentation in tf-faster-rcnn

### ISSUE THREAD

**Issue by paulcx** [endernewton/tf-faster-rcnn#190]
In tf-faster-rcnn, if I am not mistaken, only flipping has been applied as data augmentation trick, but there are other forms of data augmentation which can be used.

Specifically, how can I rotate the images in [45,90,180] degrees to feed training dataset?

**Comment by endernewton** [comment]
you can use tf function to rotate them.

**Comment by paulcx** [comment]
@endernewton How can I deal with the annotation of a rotated image then?

**Comment by endernewton** [comment]
all i can say is deal with it properly.. maybe a transformation matrix can help.

**Comment by hadi-ghnd** [comment]
@endernewton have you managed to do this yet?

**Comment by jinkos** [comment]
You won't improve the accuracy on the test set by helping your net to recognise upside down cars and dogs lying in unnatural positions? I recently did a kaggle competition that involved looking at aerial photographs of sea lions - rotation was good for that, but I don't think it will help with the COCO or VOC datasets.

**Comment by paulcx** [comment]
@jinkos It does make sense. However, the rotation should help a lot if you are trying to detect something small or some parts of object like eyes, ears.

**Comment by hadi-ghnd** [comment]
@jinkos Could you please take a look at an issue [here](https://github.com/endernewton/tf-faster-rcnn/issues/196)?
I have tried to add rotation to aerial images, but I got this error. thanks

**Comment by paulcx** [comment]
same question. @endernewton have you managed to do this yet?
